src/models/mod.rs

```diff
@@ -28,10 +28,13 @@ pub mod receipts;
 mod serializers;
 pub mod transactions;
 
-pub(crate) fn establish_connection() -> actix_diesel::Database<PgConnection> {
+pub(crate) fn get_database_credentials() -> String {
     dotenv().ok();
 
-    let database_url = env::var("DATABASE_URL")
-        .unwrap_or_else(|_| panic!("DATABASE_URL must be set in .env file"));
+    env::var("DATABASE_URL").unwrap_or_else(|_| panic!("DATABASE_URL must be set in .env file"))
```

Comment:
This is equivalent to:

```suggestion
    env::var("DATABASE_URL").expect("DATABASE_URL must be set in .env file")
```